x/gravity/keeper/attestation_handler.go

```diff
@@ -57,13 +60,7 @@ func (a AttestationHandler) handleSendToPaloma(ctx sdk.Context, claim types.MsgS
 			return sdkerrors.Wrapf(er, "Unable to log error %v, could not compute ClaimHash for claim %v: %v", addressErr, claim, er)
 		}
 
-		a.keeper.Logger(ctx).Error("Invalid SendToPaloma receiver",
-			"address", receiverAddress,
-			"cause", addressErr.Error(),
-			"claim type", claim.GetType(),
-			"id", types.GetAttestationKey(claim.GetEventNonce(), hash),
-			"nonce", fmt.Sprint(claim.GetEventNonce()),
-		)
+		liblog.FromSDKLogger(a.keeper.Logger(ctx)).WithFields("address", receiverAddress, "cause", addressErr.Error(), "claim type", claim.GetType(), "id", types.GetAttestationKey(claim.GetEventNonce(), hash), "nonce", fmt.Sprint(claim.GetEventNonce())).Error("Invalid SendToPaloma receiver")
```

Comment:
I don't care if we do this now of when we merge back into `master`, but line length should not exceed past some sensible limit.

Reply:
modified and pushed